src/framework/components/collision/trigger.js

```diff
@@ -43,9 +43,17 @@ class Trigger {
 
             const pos = entity.getPosition();
```

Comment:
these are only used in the else branch, move them there

Reply:
They are also used in the `else` statement too

Reply:
they seem to be ONLY used in the else statement?

Reply:
Yep, got my wires crossed 😓